wp-content/themes/core/components/blocks/gallery_grid/css/dialog.pcss

```diff
@@ -0,0 +1,268 @@
+        text-align: center;
+    }
+
+    .swiper-pagination-fraction,
+    .swiper-pagination-custom,
+    .swiper-container-horizontal > .swiper-pagination-bullets {
+        display: block;
+        margin-top: 10px;
+        position: static;
+    }
+
+    .swiper-container-autoheight,
+    .swiper-container-autoheight .swiper-slide {
+        text-align: center;
+    }
+
+    .site-panel--gallery__col {
+        padding-bottom: 30px;
+        padding-left: 20px;
+        padding-right: 20px;
+    }
+
+    .c-slider {
+        margin: 0 auto;
+        position: relative;
+
+        @media (--viewport-medium) {
+            margin: 0 auto;
+        }
+    }
+
+
+    .swiper-button-prev,
+    .swiper-button-next {
+        align-items: center;
+        border: 1px solid var(--color-white);
+        display: flex;
+        height: 50px;
+        justify-content: center;
+        margin-top: 0;
+        top: 50%;
+        transition: var(--transition);
+        transform: translateY(-50%);
+        width: 50px;
+        opacity: 0.5;
+
+        &:focus,
+        &:hover {
+            background-color: var(--color-white);
+            opacity: 1;
+            transition: var(--transition);
+            
+
+            &:after {
+                color: var(--color-black);
+            }
+        }
+
+        &:focus {
+            outline: none;
+        }
+    }
+
+    .swiper-button-next,
+    .swiper-container-rtl .swiper-button-prev,
+    .swiper-button-prev,
+    .swiper-container-rtl .swiper-button-next {
+        background: transparent;
+        color: var(--color-white);
+        display: none;
+
+        &:before {
+            display: none;
+        }
+
+        &:after {
+            color: var(--color-white);
+            content: var(--icon-chevron-right);
+            font-family: var(--font-family-core-icons);
+            font-size: 22px;
+        }
+
+        @media (--viewport-full) {
+            display: flex;
+        }
+    }
+
+    .swiper-button-next,
+    .swiper-container-rtl .swiper-button-prev {
+        right: -102px;
+
+        &:after {
+            content: var(--icon-chevron-right);
+        }
+    }
+
+    .swiper-button-prev,
+    .swiper-container-rtl .swiper-button-next {
+        left: -102px;
+
+        &:after {
+            content: var(--icon-chevron-left);
+        }
+    }
+
+    .swiper-pagination-fraction {
+        font-size: 16px;
+    }
+}
+
+.c-dialog[aria-hidden="true"] {
+    display: none;
+}
+
+.c-dialog__overlay {
+    background-color: #151515;
+    bottom: 0;
+    color: var(--color-white);
+    height: 100%;
+    left: 0;
+    overflow: scroll;
+    position: fixed;
+    right: 0;
+    top: 0;
+    width: calc(100% + 20px);
+    z-index: 99998;
+
+    .admin-bar & {
+        top: var(--spacer-wpadmin-mobile);
+
+        @media (--viewport-wpadmin) {
+            top: var(--spacer-wpadmin-desktop);
+        }
+    }
+}
+
+.c-dialog__overlay-inner {
+    height: calc(100% - 68px);
```

Comment:
Please create a var for this since the value is reused a few places. it's easier to adjust a single var than have to remember to update values in multiple locations that should match.